### DISCUSSION TITLE
Fast server replacement support/documentation

### DISCUSSION THREAD

**Discussion by Gaibhne** [nginx-proxy/nginx-proxy#1654] [Q&A]
I could not find any documentation on this topic, so I assume any behavior I would witness in testing is accidental and prone to change unless specified somehow.

I would like to know the path to effective replacing of servers/containers. What I mean by that is that in a scenario where I have a container A with `www.somedomain.com` running, and I would like to replace container A with container B to host `www.somedomain.com` from now, it is not clear to me if there is any supported path short of shutting down A, then starting up B. To minimize outage time, it would be much preferable if I could just give B the appropriate environment variables, start it up, and the companion container would say "hey, this guy has the same VIRTUAL_HOST/LETSENCRYPT_HOST as container A, that sounds like he wants to replace A, so lets route future incoming connections to B".

Is there any way to get that to work in the way I described ? Or if that is already the case, can we rely on that behavior not changing ?

**Comment by Gaibhne** [comment]
Apologies for submitting this to the wrong repository, I realize now that it did not belong there.

**Comment by willisweb** [comment]
So this is exactly how it works already. 
Spin up container B with the same run command as container A. 
nginx-proxy will load balance requests to both. 
Shut down container A, you’re now running on B. Not outage time

**Comment by Gaibhne** [comment]
Is that documented somewhere ? I don't want to rely on a behavior that just happens to work the way I need it at the moment.

**Reply by buchdag** [reply]
I don't believe it is, PR with doc updates are welcome !

**Comment by JKamsker** [comment]
I've got a similar problem.
What happens if my app requires some time to load up?

**Comment by amanpatel** [comment]
This is a great feature. I guess this request can be closed?  I will test it with my setup and let you know.
